editor/editor_fonts.cpp

```diff
@@ -262,10 +268,54 @@ void editor_register_fonts(Ref<Theme> p_theme) {
+	FontJapaneseBold->set_language_support_override("zh_CN", false);
+	FontJapanese->set_language_support_override("zh_TW", false);
+	FontJapaneseBold->set_language_support_override("zh_TW", false);
+
+	Ref<FontData> FontKorean = load_cached_internal_font(_font_NotoSansKR_Regular, _font_NotoSansKR_Regular_size, font_hinting, font_antialiased, true);
+	Ref<FontData> FontKoreanBold = load_cached_internal_font(_font_NotoSansKR_Bold, _font_NotoSansKR_Bold_size, font_hinting, font_antialiased, true);
+	FontKorean->set_language_support_override("ja", false);
+	FontKoreanBold->set_language_support_override("ja", false);
+	FontKorean->set_language_support_override("ko", true);
+	FontKoreanBold->set_language_support_override("ko", true);
+	FontKorean->set_language_support_override("zh_CN", false);
+	FontKoreanBold->set_language_support_override("zh_CN", false);
+	FontKorean->set_language_support_override("zh_TW", false);
+	FontKoreanBold->set_language_support_override("zh_TW", false);
+
+	Ref<FontData> FontChineseSimplified = load_cached_internal_font(_font_NotoSansSC_Regular, _font_NotoSansSC_Regular_size, font_hinting, font_antialiased, true);
+	Ref<FontData> FontChineseSimplifiedBold = load_cached_internal_font(_font_NotoSansSC_Bold, _font_NotoSansSC_Bold_size, font_hinting, font_antialiased, true);
+	FontChineseSimplified->set_language_support_override("ja", false);
+	FontChineseSimplifiedBold->set_language_support_override("ja", false);
+	FontChineseSimplified->set_language_support_override("ko", false);
+	FontChineseSimplifiedBold->set_language_support_override("ko", false);
+	FontChineseSimplified->set_language_support_override("zh_CN", true);
+	FontChineseSimplifiedBold->set_language_support_override("zh_CN", true);
+	FontChineseSimplified->set_language_support_override("zh_TW", false);
+	FontChineseSimplifiedBold->set_language_support_override("zh_TW", false);
+
+	Ref<FontData> FontChineseTraditional = load_cached_internal_font(_font_NotoSansTC_Regular, _font_NotoSansTC_Regular_size, font_hinting, font_antialiased, true);
+	Ref<FontData> FontChineseTraditionalBold = load_cached_internal_font(_font_NotoSansTC_Bold, _font_NotoSansTC_Bold_size, font_hinting, font_antialiased, true);
+	FontChineseTraditional->set_language_support_override("ja", false);
+	FontChineseTraditionalBold->set_language_support_override("ja", false);
+	FontChineseTraditional->set_language_support_override("ko", false);
+	FontChineseTraditionalBold->set_language_support_override("ko", false);
+	FontChineseTraditional->set_language_support_override("zh_CN", false);
+	FontChineseTraditionalBold->set_language_support_override("zh_CN", false);
+	FontChineseTraditional->set_language_support_override("zh_TW", true);
+	FontChineseTraditionalBold->set_language_support_override("zh_TW", true);
+
 	/* Droid Sans */
 
 	Ref<FontData> FontFallback = load_cached_internal_font(_font_DroidSansFallback, _font_DroidSansFallback_size, font_hinting, font_antialiased, true);
```

Comment:
`DroidSansFallback` probably can be removed as well, it seems to be used for CJK characters only.

Reply:
I think so, yeah. I'll update the PR in a bit

Reply:
You can then also remove the match section in https://github.com/godotengine/godot/blob/master/thirdparty/README.md#fonts

Reply:
Yeah, I'll remove those also. Actually, I see OpenSans is included too. Is that actually used anywhere?

Reply:
> Is that actually used anywhere?

It's for export templates.

Reply:
I'll leave that one be then and just remove all the DroidSans stuff